ERROR 2025-06-05 11:05:28 FaceDetModelLoader.py: 30] The model failed to load, please check the model path: face_sdk/models\face_detection\face_detection_1.0\face_detection_retina.pkl!
INFO 2025-06-05 11:05:32 FaceDetModelLoader.py: 16] Start to analyze the face detection model, model path: face_sdk/models, model category: face_detection，model name: face_detection_1.0
INFO 2025-06-05 11:05:32 BaseModelLoader.py: 34] Successfully parsed the model configuration file meta.json!
ERROR 2025-06-05 11:05:32 FaceDetModelLoader.py: 30] The model failed to load, please check the model path: face_sdk/models\face_detection\face_detection_1.0\face_detection_retina.pkl!
INFO 2025-06-05 11:05:36 FaceDetModelLoader.py: 16] Start to analyze the face detection model, model path: face_sdk/models, model category: face_detection，model name: face_detection_1.0
INFO 2025-06-05 11:05:36 BaseModelLoader.py: 34] Successfully parsed the model configuration file meta.json!
ERROR 2025-06-05 11:05:36 FaceDetModelLoader.py: 30] The model failed to load, please check the model path: face_sdk/models\face_detection\face_detection_1.0\face_detection_retina.pkl!
INFO 2025-06-05 11:05:40 FaceDetModelLoader.py: 16] Start to analyze the face detection model, model path: face_sdk/models, model category: face_detection，model name: face_detection_1.0
INFO 2025-06-05 11:05:40 BaseModelLoader.py: 34] Successfully parsed the model configuration file meta.json!
ERROR 2025-06-05 11:05:40 FaceDetModelLoader.py: 30] The model failed to load, please check the model path: face_sdk/models\face_detection\face_detection_1.0\face_detection_retina.pkl!
INFO 2025-06-05 11:13:15 FaceDetModelLoader.py: 16] Start to analyze the face detection model, model path: face_sdk/models, model category: face_detection，model name: face_detection_1.0
INFO 2025-06-05 11:13:15 BaseModelLoader.py: 34] Successfully parsed the model configuration file meta.json!
ERROR 2025-06-05 11:13:15 FaceDetModelLoader.py: 30] The model failed to load, please check the model path: face_sdk/models\face_detection\face_detection_1.0\face_detection_retina.pkl!
INFO 2025-06-05 11:13:20 FaceDetModelLoader.py: 16] Start to analyze the face detection model, model path: face_sdk/models, model category: face_detection，model name: face_detection_1.0
INFO 2025-06-05 11:13:20 BaseModelLoader.py: 34] Successfully parsed the model configuration file meta.json!
ERROR 2025-06-05 11:13:20 FaceDetModelLoader.py: 30] The model failed to load, please check the model path: face_sdk/models\face_detection\face_detection_1.0\face_detection_retina.pkl!
INFO 2025-06-05 11:13:35 FaceDetModelLoader.py: 16] Start to analyze the face detection model, model path: face_sdk/models, model category: face_detection，model name: face_detection_1.0
INFO 2025-06-05 11:13:35 BaseModelLoader.py: 34] Successfully parsed the model configuration file meta.json!
INFO 2025-06-05 11:13:35 FaceDetModelLoader.py: 34] Successfully loaded the face detection model!
INFO 2025-06-05 11:13:35 FaceAlignModelLoader.py: 16] Start to analyze the face landmark model, model path: face_sdk/models, model category: face_alignment，model name: face_alignment_1.0
INFO 2025-06-05 11:13:35 BaseModelLoader.py: 34] Successfully parsed the model configuration file meta.json!
ERROR 2025-06-05 11:13:35 FaceAlignModelLoader.py: 25] The model failed to load, please check the model path: face_sdk/models\face_alignment\face_alignment_1.0\face_landmark_pfld.pkl!
INFO 2025-06-05 11:13:40 FaceDetModelLoader.py: 16] Start to analyze the face detection model, model path: face_sdk/models, model category: face_detection，model name: face_detection_1.0
INFO 2025-06-05 11:13:40 BaseModelLoader.py: 34] Successfully parsed the model configuration file meta.json!
INFO 2025-06-05 11:13:40 FaceDetModelLoader.py: 34] Successfully loaded the face detection model!
INFO 2025-06-05 11:13:40 FaceAlignModelLoader.py: 16] Start to analyze the face landmark model, model path: face_sdk/models, model category: face_alignment，model name: face_alignment_1.0
INFO 2025-06-05 11:13:40 BaseModelLoader.py: 34] Successfully parsed the model configuration file meta.json!
ERROR 2025-06-05 11:13:40 FaceAlignModelLoader.py: 25] The model failed to load, please check the model path: face_sdk/models\face_alignment\face_alignment_1.0\face_landmark_pfld.pkl!
INFO 2025-06-05 11:13:44 FaceDetModelLoader.py: 16] Start to analyze the face detection model, model path: face_sdk/models, model category: face_detection，model name: face_detection_1.0
INFO 2025-06-05 11:13:44 BaseModelLoader.py: 34] Successfully parsed the model configuration file meta.json!
INFO 2025-06-05 11:13:44 FaceDetModelLoader.py: 34] Successfully loaded the face detection model!
INFO 2025-06-05 11:13:44 FaceAlignModelLoader.py: 16] Start to analyze the face landmark model, model path: face_sdk/models, model category: face_alignment，model name: face_alignment_1.0
INFO 2025-06-05 11:13:44 BaseModelLoader.py: 34] Successfully parsed the model configuration file meta.json!
ERROR 2025-06-05 11:13:45 FaceAlignModelLoader.py: 25] The model failed to load, please check the model path: face_sdk/models\face_alignment\face_alignment_1.0\face_landmark_pfld.pkl!
INFO 2025-06-05 11:13:51 FaceDetModelLoader.py: 16] Start to analyze the face detection model, model path: face_sdk/models, model category: face_detection，model name: face_detection_1.0
INFO 2025-06-05 11:13:51 BaseModelLoader.py: 34] Successfully parsed the model configuration file meta.json!
INFO 2025-06-05 11:13:51 FaceDetModelLoader.py: 34] Successfully loaded the face detection model!
INFO 2025-06-05 11:13:51 FaceAlignModelLoader.py: 16] Start to analyze the face landmark model, model path: face_sdk/models, model category: face_alignment，model name: face_alignment_1.0
INFO 2025-06-05 11:13:51 BaseModelLoader.py: 34] Successfully parsed the model configuration file meta.json!
ERROR 2025-06-05 11:13:51 FaceAlignModelLoader.py: 25] The model failed to load, please check the model path: face_sdk/models\face_alignment\face_alignment_1.0\face_landmark_pfld.pkl!
INFO 2025-06-05 11:27:07 FaceDetModelLoader.py: 16] Start to analyze the face detection model, model path: face_sdk/models, model category: face_detection，model name: face_detection_1.0
INFO 2025-06-05 11:27:07 BaseModelLoader.py: 34] Successfully parsed the model configuration file meta.json!
INFO 2025-06-05 11:27:07 FaceDetModelLoader.py: 34] Successfully loaded the face detection model!
INFO 2025-06-05 11:27:07 FaceAlignModelLoader.py: 16] Start to analyze the face landmark model, model path: face_sdk/models, model category: face_alignment，model name: face_alignment_1.0
INFO 2025-06-05 11:27:07 BaseModelLoader.py: 34] Successfully parsed the model configuration file meta.json!
ERROR 2025-06-05 11:27:07 FaceAlignModelLoader.py: 25] The model failed to load, please check the model path: face_sdk/models\face_alignment\face_alignment_1.0\face_landmark_pfld.pkl!
INFO 2025-06-05 11:54:35 FaceDetModelLoader.py: 16] Start to analyze the face detection model, model path: face_sdk/models, model category: face_detection，model name: face_detection_1.0
INFO 2025-06-05 11:54:35 BaseModelLoader.py: 34] Successfully parsed the model configuration file meta.json!
INFO 2025-06-05 11:54:35 FaceDetModelLoader.py: 34] Successfully loaded the face detection model!
INFO 2025-06-05 11:54:35 FaceAlignModelLoader.py: 16] Start to analyze the face landmark model, model path: face_sdk/models, model category: face_alignment，model name: face_alignment_1.0
INFO 2025-06-05 11:54:35 BaseModelLoader.py: 34] Successfully parsed the model configuration file meta.json!
ERROR 2025-06-05 11:54:35 FaceAlignModelLoader.py: 25] The model failed to load, please check the model path: face_sdk/models\face_alignment\face_alignment_1.0\face_landmark_pfld.pkl!
INFO 2025-06-05 11:54:40 FaceDetModelLoader.py: 16] Start to analyze the face detection model, model path: face_sdk/models, model category: face_detection，model name: face_detection_1.0
INFO 2025-06-05 11:54:40 BaseModelLoader.py: 34] Successfully parsed the model configuration file meta.json!
INFO 2025-06-05 11:54:40 FaceDetModelLoader.py: 34] Successfully loaded the face detection model!
INFO 2025-06-05 11:54:40 FaceAlignModelLoader.py: 16] Start to analyze the face landmark model, model path: face_sdk/models, model category: face_alignment，model name: face_alignment_1.0
INFO 2025-06-05 11:54:40 BaseModelLoader.py: 34] Successfully parsed the model configuration file meta.json!
ERROR 2025-06-05 11:54:40 FaceAlignModelLoader.py: 25] The model failed to load, please check the model path: face_sdk/models\face_alignment\face_alignment_1.0\face_landmark_pfld.pkl!
INFO 2025-06-05 11:54:44 FaceDetModelLoader.py: 16] Start to analyze the face detection model, model path: face_sdk/models, model category: face_detection，model name: face_detection_1.0
INFO 2025-06-05 11:54:44 BaseModelLoader.py: 34] Successfully parsed the model configuration file meta.json!
INFO 2025-06-05 11:54:44 FaceDetModelLoader.py: 34] Successfully loaded the face detection model!
INFO 2025-06-05 11:54:44 FaceAlignModelLoader.py: 16] Start to analyze the face landmark model, model path: face_sdk/models, model category: face_alignment，model name: face_alignment_1.0
INFO 2025-06-05 11:54:44 BaseModelLoader.py: 34] Successfully parsed the model configuration file meta.json!
ERROR 2025-06-05 11:54:44 FaceAlignModelLoader.py: 25] The model failed to load, please check the model path: face_sdk/models\face_alignment\face_alignment_1.0\face_landmark_pfld.pkl!
INFO 2025-06-05 11:55:28 FaceDetModelLoader.py: 16] Start to analyze the face detection model, model path: face_sdk/models, model category: face_detection，model name: face_detection_1.0
INFO 2025-06-05 11:55:28 BaseModelLoader.py: 34] Successfully parsed the model configuration file meta.json!
